ctdcal/__init__.py

```diff
@@ -25,12 +26,14 @@ def get_ctdcal_config():
     """
     # compile config.py and save variables to dict
     config = {}
-    with resources.path("ctdcal", "config.py") as filepath:
-        try:
-            with open(filepath, mode="rb") as f:
-                exec(compile(f.read(), filepath, "exec"), config)
-        except OSError:
-            log.error(f"Failed to load config file {filepath}")
+    resource_path = pathlib.Path(resources.files("ctdcal"))
+    config_file_path = resource_path / "config.py"
+
+    try:
+        with open(config_file_path, mode="rb") as f:
```

Comment:
Because this is now a Path object, you can just get the data in one go using [Path.read_bytes()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_bytes) like the following:
`exec(compile(config_file_path.read_bytes(), str(config_file_path), "exec"), config)`

Reply:
Applied in [1878cf8](https://github.com/SIO-ODF/ctdcal/pull/102/commits/1878cf8e13c2c8a2ea09ffd06c4a82546576bb1c).